pkg/kube-metrics/server.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2019 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kubemetrics
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+
+	kcollector "k8s.io/kube-state-metrics/pkg/collector"
+)
+
+const (
+	metricsPath = "/metrics"
+	healthzPath = "/healthz"
+)
+
+func ServeMetrics(collectors [][]kcollector.Collector, host string, port int32) {
+	listenAddress := net.JoinHostPort(host, fmt.Sprint(port))
```

Comment:
Is `fmt.Sprint` necessary?

Reply:
Yes, it expects a string and that was the cleanest solution between converting `int32` to `string.